### ISSUE TITLE
ERROR: Unknown instruction: COMPOSER

### ISSUE THREAD

**Issue by mxswar** [laradock/laradock#442]
Just clean installation:

$ git clone https://github.com/LaraDock/laradock.git
Cloning into 'laradock'...

$ cd laradock/

$ docker-compose up -d php-fpm
**Creating network "laradock_default" with the default driver
Building workspace
ERROR: Unknown instruction: COMPOSER**

$ uname -a
Linux vip 4.8.0-27-generic #29-Ubuntu SMP Thu Oct 20 21:03:13 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ docker-compose -v
docker-compose version 1.9.0, build 2585387

$ docker -v
Docker version 1.13.0-rc1, build 75fd88b

**Comment by mxswar** [comment]
well, i think that this is just docker's bug
if i remove all the comments from Dockerfiles than i can install php-fpm and other images but in any case nginx and php-fpm containers are out of service (i can't do smth like docker-compose exec nginx bash, i don't see any opened docker's proxy listener on host machine etc )

imo this is ubuntu 16.10 and docker 1.13 specific, so i'll continue to watch this issue, may be i'll be solved soon

**Comment by mxswar** [comment]
interesting but after downgrading to docker 1.12 (Ubuntu 10.04 now) the previous problem still exists:
```
$ docker-compose ps
         Name                Command       State           Ports         
------------------------------------------------------------------------
laradock_applications_1   /true           Exit 0                         
laradock_nginx_1          nginx           Exit 1                         
laradock_php-fpm_1        php-fpm         Exit 78                        
laradock_workspace_1      /sbin/my_init   Up        0.0.0.0:2222->22/tcp 
```
php-fpm just can't be started at all and i can't see any logs from it.

is anyone able to reproduce the full installation sequence of the laradock with the latest images ?



possible this is a subject for a new issue

**Comment by philtrep** [comment]
@mxswar What did you end up doing to make this work?

**Comment by abbert** [comment]
Same here with Docker version 1.13.0-rc2, build 1f9b3ef

**Comment by philtrep** [comment]
@mxswar Like you pointed out this is a known Docker issue (https://github.com/docker/docker/issues/29005). Hopefully this will be resolved soon...

**Comment by philtrep** [comment]
@mxswar @abbert Fixed it by downgrading to 1.12.3.

**Comment by LarryEitel** [comment]
WOW I was just struggling with this one on a new project! Thanks! :)

I just ripped out all the comment lines embedded in run blocks for now. :)
